grails-app/controllers/org/bbop/apollo/GroupController.groovy

```diff
@@ -74,14 +77,14 @@ class GroupController {
             // restricted groups
             def groups = dataObject.groupId ? [UserGroup.findById(dataObject.groupId)] : UserGroup.all
             def filteredGroups =  groups
+            def currentUser = User.findByUsername(dataObject.username)
             // if user is admin, then include all
             // if group has metadata with the creator or no metadata then include
-
-            if (!permissionService.isAdmin()) {
+            if (!permissionService.isUserAdmin(currentUser)) {
                 log.debug "filtering groups"
 
                 filteredGroups = groups.findAll(){
-                    it.metadata == null || it.getMetaData("creator") == (permissionService.currentUser.id as String) || permissionService.isGroupAdmin(it, permissionService.currentUser)
+                    it.metadata == null || it.getMetaData("creator") == (currentUser.id as String) || permissionService.isGroupAdmin(it, currentUser)
```

Comment:
see note above.  I'd prefer leaving it the other way. 

Also "creator" should go probably into FeatureStringEnum (will need to recmpile GWT). though its not dire just yet.